application-services/custom/fledge-export/res/configuration.toml

```diff
@@ -1,52 +1,70 @@
 [Writable]
 LogLevel = 'INFO'
+  [Writable.InsecureSecrets]
+    [Writable.InsecureSecrets.DB]
+    path = "redisdb"
+      [Writable.InsecureSecrets.DB.Secrets]
+      username = ""
+      password = ""
 
 [Service]
-BootTimeout = "30s"
-CheckInterval = "10s"
-Host = "localhost"
+HealthCheckInterval = '10s'
+Host = 'localhost'
+Port = 59780 # Adjust if running multiple examples at the same time to avoid duplicate port conflicts
 ServerBindAddr = "" # if blank, uses default Go behavior https://golang.org/pkg/net/#Listen
-Port = 50001
-Protocol = "http"
-ReadMaxLimit = 100
 StartupMsg = 'LF Edge Fledge Export Service'
-Timeout = "5s"
+RequestTimeout = '30s'
+MaxRequestSize = 0
+MaxResultCount = 0
 
 [Registry]
 Host = "localhost"
 Port = 8500
 Type = "consul"
 
-[Clients]
-  [Clients.CoreData]
-  Protocol = "http"
-  Host = "localhost"
-  Port = 48080
-
-  [Clients.Logging]
-  Protocol = "http"
-  Host = "localhost"
-  Port = 48061
+# Database is require when Store and Forward is enabled or when using Redis for the Messagebus
+[Database]
+Type = "redisdb"
+Host = "localhost"
+Port = 6379
+Timeout = "30s"
 
-[Binding]
-Type="messagebus"
-SubscribeTopic="events"
-PublishTopic=""
+[Clients]
+  [Clients.core-metadata]
+  Protocol = 'http'
+  Host = 'localhost'
+  Port = 59881
 
-[MessageBus]
-Type = "zero"
-    [MessageBus.SubscribeHost]
-        Host = "localhost"
-        Port = 5563
-        Protocol = "tcp"
-    [MessageBus.PublishHost]
-        Host = "*"
-        Port = 5566
-        Protocol = "tcp"
+# SecretStore is required when Store and Forward is enabled and running with security
+# so Database credentials can be pulled from Vault.
+# Note when running in docker from compose file set the following environment variables:
+#   - SecretStore_Host: edgex-vault
+[SecretStore]
+Type = 'vault'
+Host = 'localhost'
+Port = 8200
+Path = '/v1/secret/edgex/app-fledge-export/'
```

Comment:
Should this just be app-fledge-export/ ?

Reply:
Yup, good catch